src/yo/yo.ts

```diff
@@ -51,6 +51,12 @@ export default class Yeoman {
 			const generatorVersion: any = pkg.dependencies['yeoman-generator'];
 			const generatorMeta: any = _.pick(pkg, 'name', 'version', 'description');
 
+			//generator version can be null if package is misconfigured therefore break semver.ltr() if given null
+			//see https://github.com/SamVerschueren/vscode-yo/issues/35 for details
```

Comment:
It's not because `version` is misconfigured, it's because the `yeoman-generator` is not installed.

```
// Ignore the generator if `yeoman-generator` is not installed
if (!generatorVersion) {
... 
```